train/README.md

```diff
@@ -5,13 +5,13 @@ python3 -m venv venv && source venv/bin/activate
 # Install Torch 2.0.1
 pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
 
-# Install Dependencies, (Latest main version of huggingface is critical as its giving OOM without it, We need to use this until 4.32 is out)
-pip install accelerate==0.21.0 git+https://github.com/huggingface/transformers sentencepiece packaging ninja einops wandb
+# Install Dependencies, (huggingface version 4.32 as earlier versions results in OOM)
+pip install accelerate==0.21.0 transformers==4.32.0 sentencepiece packaging ninja einops wandb
```

Comment:
Can we also ping the other ones

Reply:
Did we try 4.32.1. ? Cause there was a tokenizer fix in that release. Lets use that if its non-problematic

Reply:
I did not because the previous comment on line 8 mentioned that 4.32.0 will fix the OOM issue so I just tried 4.32.0 only. Will try 4.32.1